src/Nox.Reference.Data.World/DataContext/WorldInfo.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Reflection;
+using AutoMapper;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using Nox.Reference.Abstractions;
+using Nox.Reference.Data.World;
+
+namespace Nox.Reference.Data;
+
+public static class WorldInfo
+{
+    private static readonly IWorldInfoContext _dbContext;
+
+    static WorldInfo()
+    {
+        var mapperConfiguration = new MapperConfiguration(cfg =>
+        {
+            cfg.AddMaps(Assembly.GetExecutingAssembly());
+        });
+        var mapper = mapperConfiguration.CreateMapper();
+        IConfiguration configuration = new ConfigurationBuilder()
+            .AddJsonFile("appsetings.json")
```

Comment:
maybe let's move appsettings.json to a constant?

Reply:
but it won't be url, it would be a file name
I mean, public const string appSettingsFileName = "appsetings.json"
I think you have a typo here, check the spelling